CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 <!-- <START NEW CHANGELOG ENTRY> -->
 
+## 0.9.5
```

Comment:
@martinRenou looks like the force push made the changelog entry appear in the diff while it shouldn't?

Reply:
You may have rebased in the middle of the release that got broken. You would need to rebase again and remove this commit.

Reply:
What was broken with the release?

Reply:
Ah it looks like it was a force push to the `main` branch? (not this PR)

Reply:
I can fix that for you if you want. I don't think you can get away from this situation with the rebase in the Github UI, you need to rebase interactively from the git CLI and remove the two commits (release and merge)

Reply:
ok I just rebased and the diff looks fine now.